srcpkgs/lxd/template

```diff
@@ -1,37 +1,29 @@
 # Template file for 'lxd'
 pkgname=lxd
-version=3.14
+version=3.19
 revision=1
 build_style=go
 go_import_path=github.com/lxc/lxd
 go_build_tags=libsqlite3
 go_package="${go_import_path}/lxd ${go_import_path}/lxc
  ${go_import_path}/lxd-p2c ${go_import_path}/fuidshift"
-hostmakedepends="pkg-config"
+hostmakedepends="pkg-config git"
 makedepends="lxc-devel acl-devel dqlite-devel"
 depends="lxc acl acl-progs rsync squashfs-tools xz dnsmasq iptables"
 short_desc="Next generation system container manager"
-maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="Apache-2.0"
 homepage="https://linuxcontainers.org/lxd"
 distfiles="https://linuxcontainers.org/downloads/lxd/lxd-${version}.tar.gz"
-checksum=409e4758cbeb43b098d0265c4ce05aeeac5ae73f8914ceb1006e6a6d89fe1fe5
+checksum=afc0b0912e5fa977007cfd97805849a3a47564eaaaa1638948081665ad7224c8
 system_groups="lxd"
 
-do_configure() {
-	# the LXD tarball packages up the required dependencies
-	ln -s "$wrksrc/dist" "$GOPATH"
-}
-
-do_build() {
-	# don't go-get the dependencies, just install with what's there
-	cd "$GOSRCPATH"
-	go_package=${go_package:-$go_import_path}
-	go install -tags "${go_build_tags}" -ldflags "${go_ldflags}" ${go_package}
-}
+_libdir="${XBPS_CROSS_BASE}/usr/lib/sqlite-replication"
+LDFLAGS="-L${_libdir} -Wl,-R${_libdir}"
+# whitelist libcap LDFLAGS (see: https://github.com/lxc/lxd/issues/6727)
+export CGO_LDFLAGS_ALLOW='-Wl,-wrap,pthread_create'
 
 post_install() {
 	vinstall scripts/bash/lxd-client 644 /usr/share/bash-completion/completions lxd
-	vlicense COPYING LICENSE
```

Comment:
Why remove this, shouldn't Apache 2.0 be vlicensed?

Reply:
nope it should not be